### PR TITLE
Update app.js Parallel import

### PR DESCRIPTION
**Pull request by yikakia**

### CODE REVIEW

**Comment by yikakia** [comment]
I was a backend developer before, this is my first line of js code. If there is something wrong, just modify it directly

**Comment by DrJKL** [comment]
Might be cleaner to use [Promise.all](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all#using_promise.all_with_async_functions).

**Comment by yikakia** [comment]
@DrJKL  thks! it does looks better

File: web/scripts/app.js [hunk 1]
```diff
@@ -1001,15 +1001,17 @@ export class ComfyApp {
 	/**
 	 * Loads all extensions from the API into the window
 	 */
-	async #loadExtensions() {
+    async #loadExtensions() {
```
Comment (M1kep) [inline web/scripts/app.js]: New indent doesn't match the rest of the file.

Comment (yikakia) [reply web/scripts/app.js]: fixed

**Comment by mcmonkey4eva** [comment]
frontend is replaced by https://github.com/Comfy-Org/ComfyUI_frontend now

**Pull request closed without merging**